spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/json-shade/java/org/springframework/boot/configurationprocessor/json/JSONObject.java

```diff
@@ -708,7 +708,7 @@ public String toString() {
 	}
 
 	/**
-	 * Encodes this object as a human readable JSON string for debugging, such as: <pre>
+	 * Encodes this object as a human-readable JSON string for debugging, such as: <pre>
```

Comment:
This class is "shaded" from third-party source and should be left as-is.

Reply:
Done.